spring-cloud-gcp-samples/spring-cloud-gcp-pubsub-binder-sample/src/main/resources/static/index.html

```diff
@@ -9,6 +9,14 @@
     <form action="/newMessage" method="post">
         Message body: <input type="text" name="messageBody" /><br/>
         Username: <input type="text" name="username" /><br/>
+
+        Throw Error? (triggers the error handler)
+        <select name="throwError">
+            <option value="false">false</option>
```

Comment:
Consider adding a "selected" attribute to the false option, so the user does not have to click in the most common scenario.